main/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsStopTimeSchema.java

```diff
@@ -67,6 +73,12 @@ public interface GtfsStopTimeSchema extends GtfsEntity {
   @CachedField
   String stopHeadsign();
 
+  @ConditionallyRequired
+  GtfsTime startPickupDropOffWindow();
+
+  @ConditionallyRequired
+  GtfsTime endPickupDropOffWindow();
```

Comment:
Is there any validator implementing the conditional required logic for these fields? If not, please remove the annotation and bring it back when the logic is implemented.